### ISSUE TITLE
liblinear.InvalidInputDataException: invalid label

### ISSUE THREAD

**Issue by baekchun** [knowledge-discovery/hw2#11]
I'm using LIBLINEAR for training and I'm getting an invalid input data exception : invalid label when I run the gender.train image(building the image works fine).

I checked the train.gz file(in /data/features directory) using zless and it was written correctly(in LIBSVM format).   

Below is the screenshot of what I got : 
<img width="845" alt="screen shot 2016-10-20 at 7 12 07 pm" src="https://cloud.githubusercontent.com/assets/21991100/19581270/c598c086-96f9-11e6-9b36-9fc52059e916.png">

**Comment by TomLippincott** [comment]
I don't see anything in the liblinear Java code (Train.java) about reading in gzipped data, I just don't think that's what it's expecting.

**Comment by baekchun** [comment]
yeah, they don't mention it there in Train.java but when I tested it locally, it worked and gave me an output of model file .. ?
